USERNS.md

```diff
@@ -1,3 +1,5 @@
+[User namespaces](https://en.wikipedia.org/wiki/Linux_namespaces#User_ID_(user)) are a kernel feature introduced in kernel version 3.8. When an unprivileged user asks the kernel to create a namespace, the kernel needs to permit that user to do so. Whether this is permitted by the kernel is controlled via a sysctl flag. There is [some history](https://madaidans-insecurities.github.io/linux.html#kernel) (albeit fairly old history) of vulnerabilities made possible by allowing this functionality for unprivileged users. Given this history, you might think we should just disable this functionality altogether. However if this functionality is disabled, then flatpak can't function without the suid bit set on the bubblewrap binary. In this scenario bubblewrap creates namespaces on behalf of the user instead of the kernel, by running as root. This means trusting bubblewrap (a significantly less battle-tested piece of software than the kernel) to run as root. However, some see this as still a preferable tradeoff (trusting one small program with root in exchange for reducing the kernel's attack surface). Ultimately we leave both options available because it's a tradeoff and neither is demonstrably preferable from a security standpoint. 
+
+Since rootless `podman` requires user namespace to function, **several essential packages like `distrobox` and `toolbx` will be removed** when using secureblue without user namespace; see [disableuserns-packages.yml](https://github.com/secureblue/secureblue/blob/live/config/common/disableuserns-packages.yml) and [disableuserns-scripts.yml](https://github.com/secureblue/secureblue/blob/live/config/common/disableuserns-scripts.yml) for more details of the changes applied.
```

Comment:
I'm not sure what this provides aside from adding "toolbox" to the list. added this instead: https://github.com/secureblue/secureblue/commit/04854499db4b9bf5557d8534ac14c59cc9ce03d3